### ISSUE TITLE
Where Are The Context Menus?

### ISSUE THREAD

**Issue by opticyclic** [jidesoft/jide-oss#17]
None of your demos have context menus showing the standard Cut/Copy/Paste/Select All menus and functionality.

**Comment by jidesoft** [comment]
No, it was not included. JDAF has this feature but not in any of JIDE component demos.

**Comment by opticyclic** [comment]
It would be great if you could add it to the demos, especially if there is a cool way of having a reusable menu across lots of different components.

**Comment by jidesoft** [comment]
Not easy. JDAF made many assumptions (which may not be the case for most use cases) and built them as foundation before it can add context menu thing.
